README.md

```diff
@@ -1,62 +1,83 @@
+[`3box-graphql`](https://github.com/3box/3box-js-graphql) is a GraphQL endpoint that allows developers to write more efficient 3Box.js getProfile() queries. Now developers can ask our API for specific common user profile fields instead of needing to return the entire profile. For example, this is useful when querying for name and image for hundreds or thousands of profiles at once.
 
-## 3Box DB
-3Box DB consists of two components: [`3box-js`](https://www.github.com/3box/3box-js) which is a client library, and [`3box-address-server`](https://www.github.com/3box/3box-address-server) which is a server. Together they are used to keep track of user data stored in [`orbit-db`](https://github.com/orbitdb/orbit-db) instances. Learn more about the [3Box architecture and how we do access control with encryption](./ARCHITECTURE.md).
+## 3Box Data Network
+The 3Box user data network consists of a few core components: the [pinning server](https://www.github.com/3box/3box-pinning-server), the [address server](https://www.github.com/3box/3box-address-server), and a [caching service](https://www.github.com/3box/3box-caching-service). Together these components are used to keep track of user data stored in [`orbit-db`](https://github.com/orbitdb/orbit-db) instances on IPFS. 
 
-### 3Box JS
-[`3box-js`](https://www.github.com/3box/3box-js) is the client side library used to read, write, and delete public and private data associated with the user. This library is used to get profile information about an address.
+We assume that end-users have a web3-compatible browser or wallet (such as MetaMask or Status) which supports personal_sign. This is how users authenticate dapps to their 3Box. Learn more about the [3Box architecture and how we do access control with encryption](./ARCHITECTURE.md).
 
-We assume that end users have a web3-compatible browser or wallet (such as MetaMask or Status) which supports personal_sign.
+### 3Box Pinning Server
+[`3box-pinning-server`](https://www.github.com/3box/3box-pinning-server) is a service operated by 3Box that pins 3Box user data on the IPFS network to ensure availability. In the future this service will become more decentralized with a network of pinning nodes, allowing others to run their own node and host their own data if desired.
 
 ### 3Box Address Server
-[`3box-address-server`](https://www.github.com/3box/3box-address-server) is a server utilizing a REST-API that is used to associate an Ethereum address with its latest 3Box address. This is what must be looked up to sync the user's data.    
+[`3box-address-server`](https://www.github.com/3box/3box-address-server) is a server utilizing a REST API that is used to associate an Ethereum address with its 3Box DID (decentralized identifier), to which all user data is addressed. This is what must be looked up to retrieve and sync the user's data. In the future this service will become more decentralized by putting this address to DID mapping on-chain.
```

Comment:
`by putting this address to DID mapping on-chain.` is not really what we want to do. We want to integrate the address -> DID mapping into the pinning node network.